audb/core/load.py

```diff
@@ -1184,7 +1179,7 @@ def load_table(
                 os.path.exists(f'{table_file}.csv')
                 or os.path.exists(f'{table_file}.pkl')
         ):
-            _load_tables(
+            cached_versions = _load_tables(
```

Comment:
Here the same as with `load_media()`, we do not need the returned value

```suggestion
            _load_tables(
```

and I would also move the definition of `cached_versions = None` before calling `_load_tables`

Reply:
I removed the `cached_files = None` line and directly pass `None` to `_load_tables()`. The return value is also not stored anymore.